### PR TITLE
fix: usage of action that uses deprecated `node12`

### LINKED ISSUE #51: `lint` job `uses node12 which is deprecated`
https://github.com/brave/kuchikiki/actions/runs/8914366941

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #51.

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -23,8 +22,4 @@ jobs:
       - name: Format
         run: cargo fmt --all -- --check
 
-      - name: Clippy
-        uses: actions-rs/clippy-check@b5b5f21f4797c02da247df37026fcd0a5024aa4d # v1.0.7
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-targets
```
Comment (hamirmahal) [inline .github/workflows/lint.yaml:30]: This pull request follows the original action's behavior of not failing when there's a linting issue.

**Comment by rillian** [comment]
Thanks for the submission! Can you please update the PR to follow local style? That is, don't reformat unrelated lines and keep the `name` tag on each step?

**Comment by hamirmahal** [comment]
> Thanks for the submission! Can you please update the PR to follow local style? That is, don't reformat unrelated lines and keep the `name` tag on each step?

You're welcome! I'll restore `name` and undo the formatting changes.

**Review by rillian — APPROVED** [review]
Looks good.

**Pull request merged**